environments/development.env

```
@@ -29,3 +29,5 @@ MB__CLIENT__TIMEOUT=10
 MB__CLIENT__API_KEY=matchbox-api-key
 MB__CLIENT__RETRY_DELAY=2
 MB__CLIENT__DEFAULT_WAREHOUSE=
+MB__CLIENT__USER=scott.mcgregor
+MB__CLIENT__EVAL_RESOLUTION=
```

Perhaps?

```suggestion
MB__CLIENT__DEFAULT_EVAL_RESOLUTION=
```

Or the same as the default resolution, which should also be populated from here. But that's a bigger PR.

Or a `st.text_input()` and it's up to you to enter the right thing. This is my favoured edit to this PR.

Really we want a `st.dropdown()` populated from a list resolutions endpoint. This is my favoured actual solution in a new ticket.

---

Add a way of setting this without an env variable

---

Done